integration/cluster_test.go

```diff
@@ -489,7 +489,7 @@ func TestTransferLeaderStopTrigger(t *testing.T) {
 
 	// issue put to one of the other member
 	kvc := toGRPC(clus.Client((oldLeadIdx + 1) % 3)).KV
-	sctx, scancel := context.WithTimeout(context.TODO(), clus.Members[oldLeadIdx].electionTimeout())
+	sctx, scancel := context.WithTimeout(context.TODO(), 10*clus.Members[oldLeadIdx].electionTimeout())
```

Comment:
does the election timeout even matter here? if not, just make it a `time.Second`?